xsd/netex_framework/netex_responsibility/netex_responsibility_support.xsd

```diff
@@ -108,6 +108,22 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="lessor"/>
 			<xsd:enumeration value="dataRegistrar"/>
 			<xsd:enumeration value="other"/>
+			<!-- DEPRECATED Upper case now lowercase  -->
```

Comment:
This has absolutely nothing to do with DeckPlan.

Reply:
You are right.... however that's minor, useful, and time saving since Nick is now on holidays for a few weeks

Reply:
If it is minor, you can make a separate pull request resolving this issue.

Reply:
@Aurige I also believe this should **not** go into next. Since next will be our major release, it would be fine to remove the deprecated values.